flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestQueue.java

```diff
@@ -193,7 +193,8 @@ private void writeAndFlushNextMessageIfPossible(final Channel channel) throws IO
 						BufferResponse msg = new BufferResponse(
 							next.buffer(),
 							reader.getSequenceNumber(),
-							reader.getReceiverId());
+							reader.getReceiverId(),
+							0);
```

Comment:
I suspect, this is changed to the actual value by the next PR, isn't it?

Reply:
yes